AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -68,8 +73,14 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
         let audioDeviceCapabilities = getSelectedAudioDeviceCapabilities()
 
         let enableAudioRedundancy = audioRedundancySwitch.isOn
+        
+        let reconnectTimeoutMs = self.reconnectTimeoutOptions[self.reconnectTimeoutPicker.selectedRow(inComponent: 0)]
```

Comment:
Nice. I appreciate that you implemented a picker in the demo for easier testing.